packages/babel-helper-define-polyfill-provider/src/types.ts

```diff
@@ -0,0 +1,120 @@
+import type { NodePath } from "@babel/traverse";
+import { types as t } from "@babel/core";
+import createMetaResolver from "./meta-resolver";
+
+type ObjectMap<T> = {
+  [k: string]: T;
```

Comment:
per my commit message, this file got counted as a deletion + addition instead of a rename bc TS uses semicolons at the end of property types, while Flow uses commas. as this file is entirely types, that meant most of the file got reformatted